app/views/dashboards/show.html.erb

```diff
@@ -12,17 +12,19 @@
     <% end %>
   </p>
 
-  <table class="table">
-    <thead>
-    <tr>
-      <th></th>
-      <% @environment.deploy_groups.sort_by(&:natural_order).each do |deploy_group| %>
-        <th><%= link_to deploy_group.name, deploy_group %></th>
-      <% end %>
-    </tr>
-    </thead>
-    <tbody>
-      <tr ng-repeat="project in projects" class="{{ project.css.tr_class }}" style="{{ project.css.style }}" project_deploy_groups/>
-    </tbody>
-  </table>
+  <div class="large-table-responsive">
+    <table class="table">
```

Comment:
ah, fancy

On Fri, Jun 17, 2016 at 2:04 PM, Robert Ikeoka <notifications@github.com>
wrote:

> In app/views/dashboards/show.html.erb
> <https://github.com/zendesk/samson/pull/1079#discussion_r67575272>:
>
> > @@ -12,17 +12,19 @@
> >      <% end %>
> >    </p>
> >
> > -  <table class="table">
> > -    <thead>
> > -    <tr>
> > -      <th></th>
> > -      <% @environment.deploy_groups.sort_by(&:natural_order).each do |deploy_group| %>
> > -        <th><%= link_to deploy_group.name, deploy_group %></th>
> > -      <% end %>
> > -    </tr>
> > -    </thead>
> > -    <tbody>
> > -      <tr ng-repeat="project in projects" class="{{ project.css.tr_class }}" style="{{ project.css.style }}" project_deploy_groups/>
> > -    </tbody>
> > -  </table>
> > +  <div class="large-table-responsive">
> > +    <table class="table">
>
> This is needed for the better fix, header that stays on screen and first
> column that stays in view.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1079/files/b0d97c64118182bb26b2fe70656e0d1864146e88#r67575272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ7nB-Ax4scrv5aKr0ahmiolkBDKpks5qMwvdgaJpZM4I3mD7>
> .
>